### ISSUE TITLE
'back' button resumes live TV channel if it had 'disconnections' during playback

### ISSUE THREAD

**Issue by kpfleming** [jellyfin-archive/jellyfin-roku-legacy#1492]
## Software Versions

- Jellyfin Server Version: 10.8.12
- Roku Client Version:  1.6.6

## Describe the bug
While playing a Live TV channel with poor signal strength, occasionally the stream will 'break' and the app will report "Error during playback." It will automatically begin playing the stream again, so when the tuner signal is recovered the audio/video will reappear. This is all fine.

However, when this happens the channel becomes the *previous* activity for the purposes of the 'back' button, so pressing 'back' just restarts the channel. If there were a large number of channel blips during the playback session, the user has to press 'back' that many times (plus 1) to return to the guide (or other location where they initiated playback of the channel).

## How To Reproduce

4. Bug occurs

## Expected behavior
The 'back' button should return to the most recent activity that is *not* the same as the current activity, which probably means that when a Live TV channel is auto-restarted it should not be added to to the activity history since it's the same channel that was already playing.

- Is server local or remote? local
- Is server connection HTTP or HTTPS? https

**Comment by candry7731** [comment]
Yes, I have experianced this too. We need to try to figure out how to quick reconnect to the stream instead of starting it over again on top of the previous activity. 

There need to be a way to "reload" the stream instead of stopping and starting again... as this takes seconds to do. VLC Player has a loop feature that will start the stream again when it fails and takes ms to start... we need to try to reproduce this logic for live tv.
